JHUGenMELAv2/MELA/interface/TMCFM.hh

```diff
@@ -238,6 +238,17 @@ extern "C" {
     double gh2w1_prime6[2]; double gh2w2_prime6[2]; double gh2w3_prime6[2]; double gh2w4_prime6[2]; // q1**4 - q2**4
     double gh2w1_prime7[2]; double gh2w2_prime7[2]; double gh2w3_prime7[2]; double gh2w4_prime7[2]; // |q1**2| * |q2**2|
 
+/*
```

Comment:
Still, better to remove.
There has to be an even number of ints. You only add one.
M_Vprime and Ga_Vprime will also change after adding W (we don't have M_V vs Ga_V in MCFM).